datahub-web-react/src/app/entity/group/AddGroupMembersModal.tsx

```diff
@@ -83,9 +83,9 @@ export const AddGroupMembersModal = ({ urn, visible, onCloseModal, onSubmit }: P
         setSelectedMembers(newUsers);
     };
 
-    const onDeselectMember = (memberUrn: string) => {
+    const onDeselectMember = (memberUrn: { key: string; label: React.ReactNode; value: string }) => {
         setInputValue('');
-        const newUserActors = selectedMembers.filter((user) => user !== memberUrn);
+        const newUserActors = selectedMembers.filter((user) => user.value !== memberUrn.value);
```

Comment:
nice fix! I would suggest a rename of `memberUrn` to just `member` since it's an object we're getting back. definitely not blocking this approval though

Reply:
Thank you for your input! Created another PR for the variable rename suggestion since the current one was merged -
https://github.com/datahub-project/datahub/pull/8529